NewForce-Cohort-7/tabloidfullstack-hollywoodgoblins
# view all posts functionality achieved

Description
User can now click "Posts" inside of the navbar to be guided to the "/posts" page to view all posts . [Ticket 03]

https://trello.com/c/aQOWVRgn

Type of change
 New feature (non-breaking change which adds functionality)

Testing Instructions
 Run git add and git commit whatever you're working on on your branch
 Run git checkout main
 Run git fetch --a
 Run git checkout HD-post-t03
 If you don't see the most recent version of my files, run git pull origin HD-post-t03
 Add all seed data by adding a new query and copying the code from the links below
 Use this to fill out tables and columns:
https://github.com/NewForce-Cohort-7/tabloidfullstack-hollywoodgoblins/blob/main/SQL/01_Tabloid_Create_DB.sql
 Use this to seed the database:
https://github.com/NewForce-Cohort-7/tabloidfullstack-hollywoodgoblins/blob/main/SQL/02_Tabloid_Seed_Data.sql
 To execute your query just press Ctrl+Shift+e inside of your query.
 Run the program by clicking the Green Arrow next to TabloidFullStack at the top.
 Navigate to TabloidFullStack/client/tabloid in your terminal
 Once there run command npm install and then npm start
 After running that command you should be brought to the sign in page. Use account [foo@bar.com](mailto:foo@bar.com) to login. (NOTE: There is no password.)
 You should now be brought to the home screen landing page that lists (Home, Posts, Logout), Click on "Posts".
You should now see all posts.
Checklist:
X My code follows the style guidelines of this project
X I have performed a self-review of my own code
X I have commented my code, particularly in hard-to-understand areas
X My changes generate no new warnings or errors
X I have added test instructions that prove my fix is effective or that my feature works
